plugins/field-slider/package.json

```diff
@@ -41,12 +41,12 @@
   "devDependencies": {
     "@blockly/dev-scripts": "^1.2.32",
     "@blockly/dev-tools": "^5.2.4",
-    "blockly": "^9.2.0",
+    "blockly": "^9.3.1",
     "chai": "^4.2.0",
     "sinon": "^9.0.1"
   },
   "peerDependencies": {
-    "blockly": "^9.2.0"
```

Comment:
I think technically this could be `^9.3.0` - but no real reason not to require the patch imo.

Reply:
That was my thought. Since this is being updated anyway, I might as well update it to the latest at that time.